### DISCUSSION TITLE
Is there any way to show fetch status?

### DISCUSSION THREAD

**Discussion by Xoffio** [nushell/nushell#7645] [Q&A] [answered]
Hi everyone!
Do you know when you do something like `wget https://github.com/nushell/nushell/archive/refs/heads/main.zip -o main.zip` and you get some status or progress output? is there any way to print that with fetch?
Thanks!

**Comment by fdncred** [comment]
There is no way to print a status with fetch that I know about. It would be nice to have a progress bar maybe behind a flag.

**Comment by Xoffio** [comment]
@fdncred  If you don't mind I would really enjoy to implement that. Should I go to Discord to discuss it with the team?

**Reply by Xoffio** [reply]
Just noticed that the output  option for `fetch` was removed [#7468](https://github.com/nushell/nushell/pull/7468) so... a status progress should be implemented on the `save` item?

**Comment by Xoffio** [comment, marked as answer]
This is now possible with the `-p` or `--progress` flag. Example:
```nu
fetch https://github.com/torvalds/linux/archive/refs/heads/master.zip | save --progress -f main.zip
```
